### PR TITLE
[ES-1125] Cherry Picked the changes of #203 , #204 and #205

### PR DESCRIPTION
**Pull request by kaifk468**

### CODE REVIEW

File: mock-esignet-integration-impl/pom.xml [hunk 1]
```diff
@@ -58,13 +58,20 @@
 			<scope>compile</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>io.mosip.esignet</groupId>
+			<artifactId>esignet-core</artifactId>
+			<version>1.2.1-SNAPSHOT</version>
+			<scope>provided</scope>
```
Comment (balaji-alluru) [inline mock-esignet-integration-impl/pom.xml:65]: Please update version to 1.4.1-SNAPSHOT

File: mock-esignet-integration-impl/pom.xml [hunk 2]
```diff
@@ -58,13 +58,20 @@
...
+			<artifactId>esignet-core</artifactId>
+			<version>1.2.1-SNAPSHOT</version>
+			<scope>provided</scope>
+		</dependency>
+
 		<dependency>
 			<groupId>io.mosip.esignet</groupId>
 			<artifactId>esignet-integration-api</artifactId>
 			<version>1.3.0</version>
 			<scope>provided</scope>
```
Comment (balaji-alluru) [inline mock-esignet-integration-impl/pom.xml:72]: Please update version to 1.4.1-SNAPSHOT

File: mock-esignet-integration-impl/pom.xml [hunk 3]
```diff
@@ -58,13 +58,20 @@
...
 		</dependency>
-		
+
 		<dependency>
 			<groupId>io.mosip.kernel</groupId>
 			<artifactId>kernel-keymanager-service</artifactId>
```
Comment (balaji-alluru) [inline mock-esignet-integration-impl/pom.xml:77]: Please update version to 1.2.0.1

File: mock-esignet-integration-impl/pom.xml (hunk 1 above)
Comment (kaifk468) [reply mock-esignet-integration-impl/pom.xml:65]: done

**Pull request merged**